sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -201,9 +201,12 @@ def response_handler(okta_response):
                 "expires_in": 60
             }
 
-            ApiClientBuilder.build(
+            client = ApiClientBuilder.build(
                 api_configuration=api_configuration,
-                okta_response_handler=response_handler)
+                id_provider_response_handler=response_handler)
+
+            # Force evaluation of the access token so that it is retrieved
+            repr(client.configuration.access_token)
```

Comment:
Have to force conversion to string to run the token code and allow the response_handler to collect the tokens. Without `repr` the code to get the token never runs and so the response_handler is never executed.